src/algorithm/context/context.rs

```diff
@@ -12,9 +15,11 @@ pub struct Context {
     /// result of calculation [steady-state-lifting-speed](design\docs\algorithm\part02\chapter_01_choose_hook.md)
     pub lifting_speed: LiftingSpeedCtx,
     /// result of calculation [ϕ2(phi) and β2(betta) coefficients](design\docs\algorithm\part02\chapter_01_choose_hook.md)
-    pub bet_phi: SelectBetPhiCtx,
+    pub select_bet_phi: SelectBetPhiCtx,
     /// result of calculation [dynamic coefficient](design\docs\algorithm\part02\chapter_01_choose_hook.md)
```

Comment:
If multiple calculation steps refers to the same md file, please split it for your requirements in terms of calculation classes.
Let's not wait for Stas for now at least, in the future, Stas will be splitting the algorithms better.